Controller/ContentTypeController.php

```diff
@@ -37,6 +38,11 @@ class ContentTypeController extends Controller
     private $contentTypeService;
 
     /**
+     * @var ContentService
```

Comment:
FQN in phpdoc please.

ctrl+shift+c on "SearchService" in PhpStorm, then ctrl+w, then paste :-)